content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -34,20 +34,14 @@ different Kubernetes components.
 
 | Feature | Default | Stage | Since | Until |
```

Comment:
I recommend adding a caption for each table using the [relevant shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#table-captions). This should improve accessibility.

Reply:
Looks like there is a k8s version of the table caption shortcode:
```
{{< table caption="Local machine solutions table that lists the tools supported by the community and the ecosystem to deploy Kubernetes." >}}
```
Still, I _think_ you have to add separate content for the actual table title.

Line 27, edit, introduction to the tables:
The following tables are a summary of the feature gates that you can set on
different Kubernetes components.

nit: clarify the table titles:
Table 1:
The following table contains feature gates for features in alpha or beta states.
Table 2:
The following table contains feature gates for graduated or deprecated features.

Reply:
Will do.